scripts/dac_setup.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+
+usage="
+Usage: $(basename "$0") [-h] [-v VERSION]
+This script takes care of adding the CUE sources from Perses as external dependencies to your DaC repo.
+/!\ It must be executed at the root of your repo.
+
+Options:
+    -h  show this help text
+    -v  version of Perses (in the form of \"X.Y.Z\") from which to retrieve the dependencies
+
+Example: 
+$ ./dac_setup.sh -v 0.42.1"
+
+function dacSetup() {
+    echo "Starting DaC setup with Perses version $VERSION"
+
+    # Create the destination folders (see https://cuelang.org/docs/concepts/packages/)
+    mkdir -p cue.mod/pkg/github.com/perses/perses/pkg/model/api/v1/
+
+    # Download  & extract sources
+    curl -L https://github.com/perses/perses/archive/refs/tags/v$VERSION.tar.gz --output sources.tar.gz
+    tar -xzf sources.tar.gz
+
+    # Copy the dependencies to the destination folder
+    cd perses-$VERSION/
+    # - plugin schemas
+    cp -r --parents cue/schemas ../cue.mod/pkg/github.com/perses/perses/
+    # - DaC utilities
+    cp -r --parents cue/dac-utils ../cue.mod/pkg/github.com/perses/perses/
+    # - CUE definitions patches
+    find "pkg/model/api/v1/" -name "*.cue" -exec cp --parents {} "../cue.mod/pkg/github.com/perses/perses/" \;
```

Comment:
I'm wondering if it wouldn't make sense to have all cue definitions in a single place instead of having them in the middle of the go package.

I'm just thinking in case one day we are providing a Java pkg, the classes won't be in the middle of the go package 

Would make more sense since you move the `schemas` folder under the `cue` folder

Reply:
This actually comes from the fact to use the `--local` flag on `cue get go` command to generate the base CUE def. I did this otherwise it's not possible to do DaC within the perses repo itself (I can explain live, would be hard by writing :D). Then to apply these patches seamlessly (= seamless merge with the generated files) they have to be located in the same place as the generated files.
I know this is not ideal, maybe we can think of intermediary files to invoke somehow.. need to think about it.

Reply:
generated files & their patches are now available at from the `cue` folder like the rest of the CUE sources